core/templates/pages/landing-pages/topic-landing-page/topic-landing-page.controller.ts

```diff
@@ -104,14 +104,15 @@ angular.module('oppia').directive('topicLandingPage', [
           };
           ctrl.$onInit = function() {
             pathArray = WindowRef.nativeWindow.location.pathname.split('/');
-            ctrl.subject = pathArray[2];
-            topic = pathArray[3];
+            ctrl.subject = pathArray[1];
+            topic = pathArray[2];
             topicData = TOPIC_LANDING_PAGE_DATA[ctrl.subject][topic];
             landingPageData = topicData.page_data;
             ctrl.topicTitle = topicData.topic_title;
             ctrl.lessons = landingPageData.lessons;
             assetsPathFormat = '/landing/<subject>/<topic>/<file_name>';
-            var pageTitle = 'Learn ' + ctrl.topicTitle + ' - Oppia';
+            var pageTitle = ctrl.topicTitle +
+              ' | Add, Subtract, Multiply and Divide | Oppia';
```

Comment:
This seems like it would need to be specific to the particular topic.

Reply:
Ah, yes! I've added this conditionally for now. Once we get the taglines for other topics as well, we can remove the conditional and generalize the code.

Reply:
I think you need to implement this as a general system from the beginning. Each topic should declare this segment in a declarative way, and when we add more topics we add more fields to the relevant constant. (In general, since we already know we are going to have multiple topics, never do one-off stuff with the idea of "we'll fix this later" -- it makes things harder to maintain.)

Reply:
I've changed it and now one simply needs to update the tagLines object and the rest would be done dynamically.